varcode/effects/effect_classes.py

```diff
@@ -617,14 +617,24 @@ def __init__(
             self,
             variant,
             transcript,
-            extended_protein_sequence):
-        aa_mutation_start_offset = len(transcript.protein_sequence)
+            extended_protein_sequence,
```

Comment:
Good idea, what do you think of the current changes? I'm a little nervous about having '*' in the aa_ref sequence since the stop codon isn't made explicit anywhere else (e.g. it's not in truncating effects).